CHANGES.rst

```diff
@@ -8,6 +8,10 @@
 Changes
 =======
 
+Version 1.7.2 (to-be-determined)
+
+- Fixes classifiers to reflect Python verisons
```

Comment:
comment:
Do we need to trigger a release? 

- I agree with doing the changelog step by step, so I agree with this.
- I wonder, since the classifiers do not actually affect where it can be installed, they are mere text... Maybe we can until we have actual content/features?

Reply:
I wouldn't trigger a release, I just put them in the changelog so they pile up with the rest of the changes that will come in the release.